ush/forecast_postdet.sh

```diff
@@ -514,7 +515,10 @@ FV3_out() {
     done
   else
     # No need to copy FV3 restart files when RUN=gfs or gefs
-    ${NCP} "${DATA}/input.nml" "${COM_ATMOS_HISTORY}/input.nml"
+    ${NCP} "${DATA}/input.nml" "${COM_CONF}/input.nml"
```

Comment:
Can we prefix these files with `ufs.` in the `COM_CONF` directory? It would help distinguish them better what they are for.

Reply:
Done.